ecosystem/manager.py

```diff
@@ -131,7 +131,10 @@ def update_badges(self):
 
         for tier in Tier.all():
             for project in self.dao.get_repos_by_tier(tier):
-                tests_passed = all(result.passed for result in project.tests_results)
+                tests_passed = True
+                for type_test in project.tests_results:
+                    if type_test.test_type == "standard":
+                        tests_passed = True is type_test.passed
```

Comment:
```suggestion
                    if type_test.test_type == "standard" and type_test.passed == False:
                        tests_passed = False
```

The `True is type_test.passed` is a bit cryptic to me, does this suggestion work for you?

Reply:
Yes it's working (I had test case for that)
It was a lint suggestion, before I coded it like this : `tests_passed = True if type_test.passed else False`

Reply:
@frankharkins Does like this works for you : https://github.com/qiskit-community/ecosystem/pull/461/commits/deac138400f94728148e64aad6d6c286c7f3970c

Reply:
Yep! Perfect